README.md

```diff
@@ -8,10 +8,58 @@ An Azure subscription management, reporting, and usage monitoring system created
+- The [Functions](https://github.com/alan-turing-institute/rctab-functions) repository contains three Azure function apps, which are also pushed to DockerHub ([usage](https://hub.docker.com/r/turingrc/rctab-usage), [status](https://hub.docker.com/r/turingrc/rctab-status), and [controller](https://hub.docker.com/r/turingrc/rctab-controller)) each release.
+- The [FastApiMsal](https://github.com/alan-turing-institute/fastapimsal) repository provides integration for Microsoft Entra ID (formerly Azure AD) within FastAPI applications.
+- The [rctab-models](https://github.com/alan-turing-institute/rctab-models) repository contains [pydantic](https://docs.pydantic.dev/latest/) models that are used by the server and client components.
+
 ## Contact Information
 
 If you need to get in touch with us, please open an issue on GitHub. See also [CONTRIBUTING.md](CONTRIBUTING.md).
 
+
+## Contributors
+
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/OscartGiles"><img src="https://avatars.githubusercontent.com/u/12784013?v=4?s=100" width="100px;" alt="Oscar T Giles"/><br /><sub><b>Oscar T Giles</b></sub></a><br />Original project author, responsible for the initial design and implementation of the API and CLI.</td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Iain-S"><img src="https://avatars.githubusercontent.com/u/25081046?v=4?s=100" width="100px;" alt="Iain Stenson"/><br /><sub><b>Iain Stenson</b></sub></a><br />Added the function apps and newer features of the API and CLI.</td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/tomaslaz"><img src="https://avatars.githubusercontent.com/u/12182911?v=4?s=100" width="100px;" alt="Tomas Lazauskas"/><br /><sub><b>Tomas Lazauskas</b></sub></a><br />Initial concept, contributed to the API codebase and database, overall project guidance and support.</td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/joseph-palmer"><img src="https://avatars.githubusercontent.com/u/22678593?v=4?s=100" width="100px;" alt="Joseph Palmer"/><br /><sub><b>Joseph Palmer</b></sub></a><br />Refined the automated deployment and front-end details pages.</td>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/pwochner"><img src="https://avatars.githubusercontent.com/u/78024695?v=4?s=100" width="100px;" alt="Pamela Wochner"/><br /><sub><b>Pamela Wochner</b></sub></a><br />Added summary emails and prepared the project for open-source.</td>
+    </tr>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/dlpbc"><img src="https://avatars.githubusercontent.com/u/17943860?v=4?s=100" width="100px;" alt="Eseoghene Ben-Iwhiwhu"/><br /><sub><b>Eseoghene Ben-Iwhiwhu</b></sub></a><br />Provided code review, contributions, and feedback on documentation.</td>
```

Comment:
```suggestion
      <td align="center" valign="top" width="14.28%"><a href="https://github.com/dlpbc"><img src="https://avatars.githubusercontent.com/u/17943860?v=4?s=100" width="100px;" alt="Eseoghene Benjamin"/><br /><sub><b>Eseoghene Benjamin</b></sub></a><br />Provided code review, contributions, and feedback on documentation.</td>
```

I believe I recall Ese saying that he has changed his surname. At least, his email address seems to confirm.